nav2_bringup/bringup/launch/nav2_tb3_simulation_launch.py

```diff
@@ -161,13 +156,13 @@ def generate_launch_description():
 
     start_rviz_cmd = ExecuteProcess(
         cmd=[os.path.join(get_package_prefix('rviz2'), 'lib/rviz2/rviz2'),
-             ['-d', rviz_config_file],
-             ['__ns:=/', namespace],
-             '/tf:=tf',
-             '/tf_static:=tf_static',
-             '/goal_pose:=goal_pose',
-             '/clicked_point:=clicked_point',
-             '/initialpose:=initialpose'],
+            ['-d', rviz_config_file],
```

Comment:
I'm going to guess that this will violate the linter checks. I think you need that space so each item lines up with the `o` in os.path.

Reply:
Yeah I'm trying to figure out why its unhappy. I originally removed it in that last commit because it was upset something about the resolution of the merge conflicts. Do you see anything clear in the merge commit that looks suspect? pep / flake8's errors arent super clear to me

Reply:
@crdelsey  any idea where this could be? 

```
IndentationError: unindent does not match any outer indentation level:
```

I've tried pretty much every variant of the short merge commit changes I can imagine and flake8 says it cant read the tb3 launch file for some reason I dont understand

Reply:
I'll pull your branch and see if I can tell. Will let you know in a few minutes.

Reply:
@SteveMacenski I pushed a commit to your branch that should fix the linter errors. It was just a couple of tiny indentation errors

Reply:
I should add ... if you are trying to fix the linter errors based on CI output ... well, that way lies madness. Colcon test hides too much information. The way I do it is cd build/nav2_bringup and then run `ctest -V`

Reply:
I was just running `ament_*` but wasnt super helpful. Thanks for figuring it out. No idea why the github IDE added a space everywhere